contracts/farms/common/RevoFarmBot.sol

```diff
@@ -88,39 +96,22 @@ contract UbeswapFarmBot is ERC20, AccessControl, Pausable {
 
     address public reserveAddress;
 
-    modifier ensure(uint256 deadline) {
-        require(deadline >= block.timestamp, "FarmBot: EXPIRED");
-        _;
-    }
-
     constructor(
         address _owner,
         address _reserveAddress,
-        address _stakingRewards,
         address _stakingToken,
         address _revoFees,
-        address _swapRouter,
-        address _liquidityRouter,
         address[] memory _rewardsTokens,
         string memory _symbol
     ) ERC20("Revo FP Token", _symbol) {
```

Comment:
I've been thinking it could be nice to have different names (even if we have the same symbol) for the different farm bots, to give wallets/UIs something slightly more convenient to go off for displaying different RFPs (rather than looking at the LPs and the names of the tokens contained by them)